common/djangoapps/xblock_django/models.py

```diff
@@ -72,3 +73,66 @@ def __unicode__(self):
             disabled_xblocks=config.disabled_blocks,
             disabled_create_block_types=config.disabled_create_block_types
         )
+
+
+class XBlockConfiguration(ConfigurationModel):
+    """
+    XBlock configuration used by both LMS and Studio, and not specific to a particular template.
+    """
+
+    KEY_FIELDS = ('name',)  # xblock name is unique
+
+    class Meta(ConfigurationModel.Meta):
+        app_label = 'xblock_django'
+
+    name = models.CharField(max_length=255, null=False, db_index=True)
+    deprecated = models.BooleanField(
+        default=False,
+        verbose_name=_('show deprecation messaging in Studio')
+    )
+
```

Comment:
Nit: Can you add a comment about `enabled` existing implictly, similar to https://github.com/edx/edx-platform/blob/master/lms/djangoapps/bulk_email/models.py#L376?